plugins/UsersManager/UsersManager.php

```diff
@@ -146,7 +145,7 @@ public static function checkPassword($password)
 
         if (!self::isValidPasswordString($password)) {
             throw new Exception(Piwik::translate('UsersManager_ExceptionInvalidPassword', array(self::PASSWORD_MIN_LENGTH,
```

Comment:
A test in `Piwik\Plugins\UsersManager\tests\Integration\UsersManagerTest` fails now. You can likely execute it with `./console tests:run plugins/UsersManager/tests/Integration/UsersManagerTest.php` but you might have to setup tests see

* https://github.com/piwik/piwik/blob/master/tests/README.md#phpunit-tests
* http://developer.piwik.org/guides/tests-php